ui/user/src/lib/components/messages/Message.svelte

```diff
@@ -423,12 +423,22 @@
 			{#each msg.fields as field}
 				<div class="flex flex-col gap-1">
 					<label for={field.name} class="mt-1 text-sm font-medium">{field.name}</label>
-					<input
-						class="rounded-lg bg-white p-2 outline-hidden dark:bg-gray-900"
-						type={field.sensitive ? 'password' : 'text'}
-						name={field.name}
-						bind:value={promptCredentials[field.name]}
-					/>
+					{#if field.options}
```

Comment:
new ui components to handle field with options. For now, it is buttons but we can customize it to be dropdowns